### ISSUE TITLE
Which wxWidgets commit is used in a Phoenix release

### ISSUE THREAD

**Issue by ChrisEt** [wxWidgets/Phoenix#1713]
Could we include the commit hash of wxWidgets that was used for building a specific release somewhere?

**Reason**
I am currently investigating a segfault in `DragImage.BeginDrag()` under GTK. I know I can find all sources in the PyPi package, but it would be a real help to be able to look at the commit history in Git. It took me some time to figure out that **wxPython 4.0.7.post2** was built from **wxWidgets Git branch WX_3_0_BRANCH from 2019-10-20** (commit [3b6a9f7](https://github.com/wxWidgets/wxWidgets/commit/3b6a9f773a58365a3049d476e2858c355c1267d1))

I suggest that the build script reads the working copy's version (it could also check for local modifications) and writes this to a "release notes" section somewhere.

What do you guys think?

**Comment by swt2c** [comment]
Seems reasonable.  Although you can always just browse to Phoenix/ext and then select the tag (e.g., wxPython-4.0.7.post2) you are interested in.

**Comment by ChrisEt** [comment]
Yes, you are right, the GitHub tag includes this information. I didn't see that.

Thanks a lot, you can close this issue (or keep it open if you like the idea with the build script) 👍

**Comment by swt2c** [comment]
I think it's a good idea to record that somewhere anyway...
